src/extension.ts

```diff
@@ -1008,6 +1010,33 @@ export async function activate(context: vscode.ExtensionContext) {
     }
   });
 
+  registerIDFCommand("cmakeListsEditor.start", async (fileUri: vscode.Uri) => {
+    if (!fileUri) {
+      return;
+    }
```

Comment:
Since this command is not expected to be called directly by user, maybe we can just disable it from `package.json` and just add this handler.

Reply:
The thing is, command is provided as right click option in the explorer/context and it needs a command title and command entry otherwise an error is shown.

Reply:
Ah! I remember now... maybe we can [do something like this](https://github.com/espressif/vscode-esp-idf-extension/blob/f0dc28127f8d17cd51a42d7d3b9440f868c56957/src/extension.ts#L1147) to inform the user that we can't launch the command directly ...